test/units/util.sh

```diff
@@ -110,3 +110,48 @@ create_dummy_container() {
     cp -a /testsuite-13-container-template/* "$root"
     coverage_create_nspawn_dropin "$root"
 }
+
+# Wait until file appears
+wait_for_file() {
+    local timeout=60
+    local file
+
+    if [[ $1 == '-t' ]]; then
+        timeout="$2"
+        shift 2
+    fi
+
+    file="$1"
+
+    timeout "$timeout" bash -c "while [[ ! -e $file ]]; do sleep .5; done"
+}
+
+# Wait until command succeeds
+wait_until() {
```

Comment:
Hmm, this can simply be a wrapper for `wait_while`?

Reply:
It could be (and `wait_for_file` too), but it's not worth it. It would still have to parse the potential timeout argument to get the command, hence there'd be very little code saved.